plugins/techdocs/src/reader/transformers/transformer.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type Transformer = (dom: Element) => Element;
```

Comment:
hm the Element type isn't qualified in this file; you probably should make it `JSX.Element` for clarity

Reply:
When updating this to `JSX.Element` it caused some issues in the files that use `Transformer`, from looking at the previous code location `Element` was also not qualified there. 

I'd prefer to keep this as is for now and create minimal changes, especially since I am still new to typescript and would need to look into the nuances of `JSX.Element` `HTMLElement` `Element` etc.

What do you think?